storybook/storybook-docs/src/color.stories.mdx

```diff
@@ -0,0 +1,35 @@
+import { Meta } from "@storybook/blocks";
+import { ColourPalette } from "./components/ColourPalette";
+
+<Meta title="Docs/Assets/Colour" />
+
+# Colour
+
+The basic colours of Amsterdam are white, red, and black.
+We have 8 additional colours and 3 shades of grey.
+
+<ColourPalette>
+  <ColourPalette.Section>
+    <ColourPalette.Tile name="white" rgb="#ffffff" />
```

Comment:
Can we get these values from the design tokens? So we don't have to maintain them in 2 places.

Reply:
Certainly!

The page even reloads automatically when updating a colour token, pretty amazing.